src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -654,6 +654,9 @@ private void extractNativeLibs(String classPath) throws IOException {
         for (String jar : jars) {
             FileOps.extractFilesFromJar("." + getStaticLibraryFileExtension(), Path.of(jar),
                     libPath, getTargetSpecificNativeLibsFilter());
+            FileOps.extractFilesFromJar(".o" , Path.of(jar),
```

Comment:
Use `getObjectFileExtension()` instead of `o`.

Reply:
Extract `Path.of(jar)` and `getTargetSpecificNativeLibsFilter()` to variables, as they are used twice.

Reply:
> Use `getObjectFileExtension()` instead of `o`.

That would then fail on Windows, as that hard-coded uses ".obj" in Substrate.

Reply:
Now that #718 has been integrated, this can be refactored to include all possible file extensions, creating a list before the jar with them.